arangod/IResearch/IResearchView.cpp

```diff
@@ -1021,20 +1019,17 @@ arangodb::Result IResearchView::updateProperties(arangodb::velocypack::Slice con
     SCOPED_LOCK_NAMED(mutex, mtx);
 
     // check link auth as per https://github.com/arangodb/backlog/issues/459
-    if (!arangodb::ExecContext::current().isSuperuser()) {
```

Comment:
If the user is a superuser then the user should have access to the collection. Therefore removing this short-cut as Superuser is not well-defined.